examples/montecarlo.jl

```diff
@@ -24,7 +24,8 @@ function montecarlo(mc_eval::Function, mc_monitor::Function, reps, n_returns, po
     end
 
     if mod(pernode,pooled) != 0
-        pooled = div(pernode, commsize-1)
+        pooled = Int64(div(pernode, commsize-1))
```

Comment:
What happens is that pernode is a Float64, because it is computed using /.
There very well may be some changes that would be worth making, I'm quite a
novice with Julia. Any suggestions are welcome.
Cheers,
M.

On Thu, Oct 22, 2015 at 4:46 AM, Erik Schnetter <notifications@github.com>
wrote:

> In examples/montecarlo.jl
> <https://github.com/JuliaParallel/MPI.jl/pull/84#discussion_r42706661>:
>
> > @@ -24,7 +24,8 @@ function montecarlo(mc_eval::Function, mc_monitor::Function, reps, n_returns, po
> >      end
> >
> >      if mod(pernode,pooled) != 0
> > -        pooled = div(pernode, commsize-1)
> > +        pooled = Int64(div(pernode, commsize-1))
>
> What I wanted to suggest is to calculate pernode differently so that its
> type remains integer. It's strange to calculate it as floating-point
> quantity.
>
> I was just trying to figure out why the Int64 conversion is necessary.
> Since it does seem necessary, and not through your changes, I don't want to
> request you to change things.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaParallel/MPI.jl/pull/84/files#r42706661>.
>